connectors-api/src/main/ruby/connectors_errors.rb

```diff
@@ -6,6 +6,8 @@
 
 require 'java'
 
+require 'active_support/core_ext/string'
```

Comment:
Yep. Nothing had made use of DocumentsError yet, which means that `.truncate` hadn't been run yet. When I added the single spec for ConnectorsMonitor, I hit a NoMethodError, because `String.truncate` isn't a ruby function, it comes from active support.

We're probably going to see a lot of things like this due to the lack of explicit `require ___` in our ent-search code.